.gitignore

```diff
@@ -1,2 +1,4 @@
 /dist/
 /dist-newstyle/
+.ghc.environment.*
+TAGS
```

Comment:
I don't really mind, but in general I do think it's bad form to pollute `.gitignore` with all kinds of extra stuff. `TAGS` is specific to your editor setup and should be in your private `.git/info/exclude` or `$HOME/.config/git/ignore`, not here.

Reply:
OK, sure.